archetypes/multilingual/interfaces.py

```diff
@@ -1,5 +1,10 @@
 # -*- coding: utf-8 -*-
 from plone.app.multilingual.interfaces import ITranslatable
+from zope.interface import Interface
+
+
+class IArchetypesMultilingualInstalled(Interface):
```

Comment:
I guess you meant inherit from `plone.app.multilingual.interfaces.IPloneAppMultilingualInstalled`?